neurom/apps/morph_stats.py

```diff
@@ -112,7 +116,14 @@ def extract_dataframe(neurons, config):
     if 'neuron' in config:
         del config['neuron']
 
-    stats = {nrn.name: extract_stats(nrn, config) for nrn in neurons}
+    with multiprocessing.Pool(n_workers) as pool:
+        stats = {nrn if isinstance(nrn, str) else nrn.name:
+                 stat for nrn, stat in tqdm(zip(neurons,
+                                                pool.imap(partial(extract_stats, config=config),
+                                                          neurons)
+                                                ), total=len(neurons)
+                                            )
```

Comment:
Could you use dask instead. It should be something like:
```python
dask_bag.from_sequence(neurons, npartitions=n_workers).map(extract_stats, config=config).compute()
```

Reply:
Is that extra dependency needed?

Reply:
Alexis is trying and saying it's not working well. Let's stay on Pool.

Reply:
no I'm going to stick to multiprocess, it does the job for now

Reply:
> Alexis is trying and saying it's not working well.

Weird, what kind of problem is there?  Is it a dask related thing?

Reply:
For my setting, it needs to have OPENBLAS_NUM_THREADS=1, and it is much slower. I still have to undersand how this dask works, but it is not so easy to have it working out of the box so for here, I guess standard multiprocess is good enough. The main slow part was loading neurons I think.

Reply:
Thanks for the info - appreciated.